spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/security/OAuthSecurityConfiguration.java

```diff
@@ -43,15 +45,23 @@ protected void configure(HttpSecurity http) throws Exception {
 		http.antMatcher("/**")
 		.authorizeRequests()
 			.antMatchers(
-					"/security/info**", "/login**", "/"+ UiController.WEB_UI_INDEX_PAGE_NAME +"/logout-success-oauth.html",
-					"/"+ UiController.WEB_UI_INDEX_PAGE_NAME +"/styles/**", "/"+ UiController.WEB_UI_INDEX_PAGE_NAME +"/images/**",
-					"/"+ UiController.WEB_UI_INDEX_PAGE_NAME +"/fonts/**", "/"+ UiController.WEB_UI_INDEX_PAGE_NAME +"/lib/**").permitAll()
+					"/security/info**", "/login**", StringUtils.arrayToCommaDelimitedString(appendIndexPageRoute("/logout-success-oauth.html",
+							"/styles/**", "/images/**", "/fonts/**", "/lib/**"))).permitAll()
 			.anyRequest().authenticated()
 		.and().httpBasic()
-		.and().logout().logoutSuccessUrl("/"+ UiController.WEB_UI_INDEX_PAGE_NAME +"/logout-success-oauth.html")
+		.and().logout().logoutSuccessUrl(UiController.WEB_UI_INDEX_PAGE_ROUTE + "/logout-success-oauth.html")
 		.and().csrf().disable();
 	}
 
+	private String[] appendIndexPageRoute(String... patterns) {
```

Comment:
Will change as well, but this is not technically the "index" page route. It's the whole *path* to the UI app.